.pre-commit-config.yaml

```diff
@@ -64,8 +64,7 @@ repos:
       - id: black
         name: Format code
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 3.9.2
+  - repo: https://github.com/asottile/yesqa
+    rev: v1.2.3
     hooks:
-      - id: flake8
-        name: Check PEP8
```

Comment:
Can we keep flake8 in pre-commit?

I personally found it very useful to catch errors before pushing

Reply:
but how the pre-commit bot works it fails on flake8, does it send a correction to the PR?

Reply:
It doesn't, but it isn't expected that all pre-commit hooks send corrections. For example

```
      - id: check-yaml
      - id: check-docstring-first
      - id: check-executables-have-shebangs
      - id: check-toml
      - id: check-case-conflict
      - id: check-added-large-files
```